mlora.py

```diff
@@ -212,9 +239,15 @@ def train(config: Dict[str, any], llm_model: mlora.LLMModel, dispatcher: mlora.D
                 all_optimizer[lora.adapter_name_].step()
 
         if step_cnt % config["save_step"] == 0:
-            mlora.save_lora_model(llm_model, config, f"{step_cnt}")
+            if "mixlora" in config and config["mixlora"]:
+                mlora.save_mixlora_model(llm_model, config, f"{step_cnt}")
```

Comment:
just use one interface to save model, use the llm_model's type to chose different method save weight. or move the save interface to llm_model's base class.